react/utils/runtime.graphql

```diff
@@ -1,5 +1,9 @@
 query Runtime ($page: String!, $production: Boolean!, $locale: String!, $renderVersion: String!) {
   page(page: $page, production: $production, locale: $locale, renderVersion: $renderVersion) {
     messagesJSON
+    pagesJSON
```

Comment:
How was this working before?!

Reply:
😱

Reply:
I was also shocked! 
Guess no one was using updateRuntime on anything critical.